src/pages/ChooseArea.svelte

```diff
@@ -5,17 +5,20 @@
   import { Map } from "maplibre-gl";
   import { onMount } from "svelte";
   import DefaultButton from "../lib/govuk/DefaultButton.svelte";
+  import ErrorMessage from "../lib/govuk/ErrorMessage.svelte";
   import FormElement from "../lib/govuk/FormElement.svelte";
   import Radio from "../lib/govuk/Radio.svelte";
   import SecondaryButton from "../lib/govuk/SecondaryButton.svelte";
   import "maplibre-gl/dist/maplibre-gl.css";
-  import authoritiesUrl from "../../assets/authorities.geojson?url";
+  import { getAuthoritiesData } from "../lib/common/data_getter";
   import FileInput from "../lib/common/FileInput.svelte";
   import About from "../lib/sidebar/About.svelte";
   import { bbox } from "../maplibre_helpers";
   import type { Schema } from "../types";
 
   let showAbout = false;
+  const params = new URLSearchParams(window.location.search);
+  let errorMessage: string = params.get("error")!;
```

Comment:
The `!` insists that this isn't null, but that's not true here. `params.get("error") || ""` would work, since we're using the empty string to mean none